hw/ip_templates/otp_ctrl/rtl/otp_ctrl.sv.tpl

```diff
@@ -1593,8 +1593,8 @@ end
   `endif
   if (`PRIM_DEFAULT_IMPL == prim_pkg::ImplGeneric) begin : gen_reg_we_assert_generic
```

Comment:
`PRIM_DEFAULT_IMPL == prim_pkg::ImplGeneric` that code should be removed at all, right?